### ISSUE TITLE
src-alt doesn't work for remote images

### ISSUE THREAD

**Issue by martinhearn** [mosbth/cimage#146]
Hi 
I can't get `?src-alt=dummy` to work for remote images when there is a failure (403,404 etc) - Instead an exception is thrown. I couldn't see a way of catching the error and displaying the dummy image - am I missing something?
Thanks!
Martin

**Comment by mosbth** [comment]
Thats right, [`alt-src` does not work together with remote images](https://cimage.se/doc/features-and-options#option-src). It was a choice made during implementation of that option. Implementing `alt-src` for remote images required some additional effort and I did choose not to do so.

I can implement it to support remote images, if there is a need for it.

**Comment by martinhearn** [comment]
Thanks @mosbth - it would be a useful feature. If i get a chance I will take a look...

**Comment by mosbth** [comment]
I'll add it to the backlog for now. It was a known limitation when I added the option for `src-alt`.
